.github/CODEOWNERS

```diff
@@ -6,17 +6,17 @@ po/de.po @ronaldfw @fechu
 po/el.po @Mandragorian
 po/es.po @deavid @vzz1x2
 po/fa.po @DannyRavi @hamidrezakp @moaminsharifi @mehrad77 @javad-jafari
-po/fr.po @sakex @lb034582341
-po/id.po @tjonganthony @nurlitadf
-po/it.po @detro @nicomazz @bznein
+po/fr.po @sakex
+po/id.po
```

Comment:
Having no owners for this translation doesn't seem ideal, though if there really aren't any then should it be removed from the file entirely?

Reply:
I agree, there was only an initial effort to set it up and then nothing. I'll go ahead and remove it all.